static/app/views/app/index.tsx

```diff
@@ -246,16 +246,16 @@ function App({children, params}: Props) {
         <RouteAnalyticsContextProvider>
           <OrganizationContextProvider>
             <AsyncSDKIntegrationContextProvider>
-              <GlobalDrawer>
-                <GlobalFeedbackForm>
+              <GlobalFeedbackForm>
```

Comment:
Had to lift the feedback from context one level so that the drawer component could access it